addons/mrp/views/mrp_routing_views.xml

```diff
@@ -49,14 +49,12 @@
                             </group>
                         </group>
                         <notebook>
-                            <page string="Description" name="description">
-                                <field name="note"/>
-                            </page>
                             <page string="Work Sheet" name="worksheet">
                                 <group>
                                     <field name="worksheet_type" widget="radio"/>
-                                    <field name="worksheet" help="Upload your PDF file." widget="pdf_viewer" attrs="{'invisible':  [('worksheet_type', '=', 'google_slide')]}"/>
-                                    <field name="worksheet_google_slide" placeholder="Google Slide Link" widget="embed_viewer" attrs="{'invisible':  [('worksheet_type', '=', 'pdf')]}"/>
+                                    <field name="worksheet" help="Upload your PDF file." widget="pdf_viewer" attrs="{'invisible':  [('worksheet_type', '!=', 'pdf')]}"/>
+                                    <field name="worksheet_google_slide" placeholder="Google Slide Link" widget="embed_viewer" attrs="{'invisible':  [('worksheet_type', '!=', 'google_slide')]}"/>
+                                    <field name="note" placeholder="..." attrs="{'invisible':  [('worksheet_type', '!=', 'text')]}"/>
```

Comment:
is `placeholder="..."` what you want?

Reply:
Well, It is enough self-explained without a placeholder, I am going to remove it.